src/main/scala/is/hail/rvd/OrderedRVDPartitioner.scala

```diff
@@ -161,10 +155,9 @@ class OrderedRVDPartitioner(
 
   def copy(
     kType: TStruct = kType,
-    rangeBounds: IndexedSeq[Interval] = rangeBounds,
-    allowedOverlap: Int = allowedOverlap
+    rangeBounds: IndexedSeq[Interval] = rangeBounds
```

Comment:
Why is allowedOverlap deleted from the copy method?

Reply:
Because `allowedOverlap` is no longer a data-member of the class.